### PR TITLE
add Min-SNR loss to Controlnet flax train script 

### PR DESCRIPTION
**Pull request by yiyixuxu**
(a very short) wandb experiment here 
https://wandb.ai/yiyixu/train_controlnet_flax?workspace=user-yiyixu

### CODE REVIEW

File: examples/controlnet/train_controlnet_flax.py [hunk 1]
```diff
@@ -806,6 +812,17 @@ def main():
     validation_rng, train_rngs = jax.random.split(rng)
     train_rngs = jax.random.split(train_rngs, jax.local_device_count())
 
+    def compute_snr(timesteps):
```
Comment (sayakpaul) [inline examples/controlnet/train_controlnet_flax.py:815]: Could you also add a comment on the reference like here?

https://github.com/huggingface/diffusers/blob/ce144d6dd05c4c588d9f2970301ace70eee16d5d/examples/text_to_image/train_text_to_image.py#L497

**Review by sayakpaul — APPROVED** [review]
Thank you so much for doing this quickly. 

Let's maybe also update the README mentioning this feature briefly?

**Comment by kashif** [comment]
thank you so much, this is great! Can you kindly also remove the line from the doc that this is only for Pytorch: https://github.com/huggingface/diffusers/blob/main/docs/source/en/training/text2image.mdx#training-with-min-snr-weighting

**Comment by yiyixuxu** [comment]
@kashif I will remove that line once we updated the flax script for text2img (I only updated controlnet in this PR)

**Review by sayakpaul — APPROVED** [review]
Looks pretty nice!

File: examples/controlnet/train_controlnet_flax.py [hunk 2]
```diff
@@ -328,11 +335,8 @@ def parse_args():
     parser.add_argument(
         "--report_to",
         type=str,
-        default="tensorboard",
-        help=(
-            'The integration to report the results and logs to. Supported platforms are `"tensorboard"`'
-            ' (default), `"wandb"` and `"comet_ml"`. Use `"all"` to report to all integrations.'
-        ),
+        default="wandb",
```
Comment (pcuenca) [inline examples/controlnet/train_controlnet_flax.py:338]: Did we mean to change the default here? (Just asking)

Comment (yiyixuxu) [reply examples/controlnet/train_controlnet_flax.py:338]: I just didn't want it to default to something that doesn't exist - cause wandb is the only method I implemented 😅 (and I don't think the other flax training scripts implemented any of these logging methods)

Comment (pcuenca) [reply examples/controlnet/train_controlnet_flax.py:338]: lol, fair enough :)
